{"uuid":"1182a97f-fb26-4a4a-930b-1f7945df521f","name":"Scenario Outline: The user getting error message with invalid python code for Insertion page","children":["36702788-13c4-4e63-9c45-1f984b7493be"],"befores":[{"name":"AppHooks.ApplicationHooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705693734771,"stop":1705693734771}],"afters":[{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705693734920,"stop":1705693734921},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705693735073,"stop":1705693735073},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705693735518,"stop":1705693735518},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705693735518,"stop":1705693735518},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705693735839,"stop":1705693735839},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705693736053,"stop":1705693736053},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705693736091,"stop":1705693736091},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705693736096,"stop":1705693736097}],"start":1705693734770,"stop":1705693736097}